### PR TITLE
feat: navigation menu icons

### PR DESCRIPTION
**Pull request by yanzhihong23**

### CODE REVIEW

**Review by Jack-Works — APPROVED** [review]

File: packages/icons/general/MenuPersonas.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import { createIcon } from '../utils'
+export const MenuPersonasIcon = createIcon(
+    'MenuPersonasIcon',
+    <>
```
Comment (guanbinrui) [inline packages/icons/general/MenuPersonas.tsx:4]: Use `<g>` to remove the lost key warning.

File: packages/icons/general/MenuLabsActive.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { createIcon } from '../utils'
+export const MenuLabsActiveIcon = createIcon(
+    'MenuLabsActiveIcon',
+    <>
```
Comment (guanbinrui) [inline packages/icons/general/MenuLabsActive.tsx:4]: Use `<g>` to remove the lost key warning.

File: packages/icons/general/MenuLabs.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import { createIcon } from '../utils'
+export const MenuLabsIcon = createIcon(
+    'MenuLabsIcon',
+    <>
```
Comment (guanbinrui) [inline packages/icons/general/MenuLabs.tsx:4]: Use <g> to remove the lost key warning.

File: packages/icons/general/MenuPersonasActive.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import { createIcon } from '../utils'
+export const MenuPersonasActiveIcon = createIcon(
+    'MenuPersonasActiveIcon',
+    <>
```
Comment (guanbinrui) [inline packages/icons/general/MenuPersonasActive.tsx:4]: Use <g> to remove the lost key warning.

**Comment by guanbinrui** [comment]
@yanzhihong23 Use `<g />` as the root node of those SVG icon components. Otherwise, I will get the lost key warming in some edge cases.

**Review by Jack-Works — APPROVED** [review]

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**